packages/core-utils/src/logic-utils.errors.ts

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { switchCase } from './logic-utils';
```

Comment:
lets add at least a couple of positive type test cases in the logic-utils.types.ts

Reply:
ah, I am satisfied with that :)

Reply:
How would those cases differ from what is in [packages/core-utils/src/logic-utils.test.ts](https://github.com/DataBiosphere/terra-ui/blob/dev/packages/core-utils/src/logic-utils.test.ts)?

Reply:
they wouldn't really, which is why I agreed with Nick's point :)

Reply:
Next time I will refresh the conversation before commenting ;)